src/apps/shared/searching.py

```diff
@@ -0,0 +1,19 @@
+from functools import reduce
+
+from sqlalchemy import Unicode, or_
+
+__all__ = ["Searching"]
+
+
+class Searching(object):
+    search_fields: list = list()
```

Comment:
In other places, you define the list as `[]`. What is different here?

Reply:
Also if you define the list type annotation I think it would be great if you will add the item type also that will be placed in that list.

You have the `search_field.cast(Unicode()).ilike(f"%{search_term}%")` and I have no idea what `search_field` is :)

Reply:
Another thing. Remove the inheriting from the object (deprecated since python 3).